### PR TITLE
Add Left Padding and Padding to Max Length

### PR DESCRIPTION
**Pull request by apaniukov**
CVS-131898
CVS-132172

- Support left padding
- Support padding to `max_length`
- Change `max_length` during CLI conversion
- Detokenizer skip special tokens by default
- Update tests pass-rate calculation - exclude skipped tests
- HF tokenizer object attributes for padding and truncation now have priority over values from `tokenizer.json` for better alignment

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
can we add tests?

**Comment by apaniukov** [comment]
> can we add tests?

Working on it

File: README.md [hunk 1]
```diff
@@ -630,92 +630,92 @@ This report is autogenerated and includes tokenizers and detokenizers tests. The
...
-      <td >81</td>
+      <td >94.38</td>
+      <td >89</td>
     </tr>
     <tr>
       <td >WordPiece</td>
       <td >rasa/LaBSE</td>
-      <td >90.12</td>
-      <td >81</td>
+      <td >80.00</td>
```
Comment (ilya-lavrenov) [inline README.md:705]: is it OK that pass rate is decreased?

Comment (apaniukov) [reply README.md:705]: There is no real degradation for this tokenizer. For some reason, numpy sometimes lets elementwise comparison for differently shaped tensors. There was one test (from `test_hf_wordpiece_tokenizers_multiple_strings`) that successfully passed `np.all((ov_result := ov_tokenized[output_name]) == hf_result)` for `(6, 1)` and `(6, 0)` shaped tensors, resulting in 5/8 pass rate (should be 4/8). I added a shape check and new test cases, so in this PR the pass rate is 16/32. That is why the total pass rate is lower.

**Review by ilya-lavrenov — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by ilya-lavrenov** [comment]
Looks like current OV tokenizers master is broken against latest OV master.
Two other PRs to OV tokenizers master also shows access violation issue.

**Pull request merged**